src/assets/sass/protocol/components/_modal.scss

```diff
@@ -81,7 +81,7 @@ html.mzp-is-noscroll {
     .mzp-c-modal-button-close {
         @include image-replaced;
         background: transparent url('#{$image-path}/icons/close-white.svg') center center no-repeat;
-        @include background-size(16px 16px);
+        @include background-size(30px 30px);
```

Comment:
Thank you for the pull request. Could you please make this `20px 20px` to match the same size the icons on the menu and details components.

Reply:
Yes, definitely.

Reply:
@stephaniehobson 
The requested changes have been made, kindly review.
Thanks